frontend/src/index.css

```
@@ -2,39 +2,39 @@
 @tailwind components;
 @tailwind utilities;
 
-:root { --bg:#0b0c0e; --surface:#151719; --surface-2:#1b1d1f; --surface-raised:#1e2124; --line:#4a443a; --line-strong:#6a6255; --field:#0e1011; --text:#eee9dc; --muted:#9d9a90; --purple:#b6975a; --purple-strong:#d7ba78; --positive:#79c99a; --negative:#d88786; --warning:#d7aa61; --info:#82b8c4; --brass:#b6975a; --brass-bright:#e1c47e; --ink:#08090a; --graphite:#202224; --flare:#e3be65; }
+:root { --bg:#0b0c0e; --surface:#151719; --surface-2:#1b1d1f; --surface-raised:#1e2124; --line:#4a443a; --line-strong:#6a6255; --field:#0e1011; --text:#eee9dc; --muted:#9d9a90; --purple:#b6975a; --purple-strong:#d7ba78; --positive:#79c99a; --negative:#d88786; --warning:#d7aa61; --info:#82b8c4; --brass:#b6975a; --brass-bright:#e1c47e; --ink:#08090a; --graphite:#202224; --flare:#e3be65; --fs-micro:11px; --fs-small:12px; --fs-body:13px; --fs-panel:15px; --fs-metric:20px; --fs-display:26px; --font-mono:ui-monospace,SFMono-Regular,Consolas,"Liberation Mono",monospace; }
```

Stylelint (17.14.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)